index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>サイト制作課題</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@400;700&display=swap"
+        rel="stylesheet" />
+</head>
+
+<body>
+    <header class="header">
+        <h1 class="header-logo-wrapper">
+            <img class="header-logo" src="./img/logo.svg" alt="Grand Players">
+        </h1>
+    </header>
+
+    <section class="hero">
+
+        <div class="hero-container">
+            <div class="hero-texts">
+                <p class="hero-main-copy">
+                    BE <br>
+                    A <br>
+                    <span class="hero-main-copy-ace">
+                        "GRAND" <br>
```

Comment:
```suggestion
                        “GRAND” <br>
```
ここの修正は任意で大丈夫です。
細かいところですが。
figmaから文章のコピーも可能なのでそうするとミスが減ります。
![image](https://github.com/user-attachments/assets/f3a8535b-f0e7-44c5-a235-b15aa66e9d08)